sharepoint/sharepoint-ps/sharepoint-online/Get-SPOAppBillingPolicies.md

```diff
@@ -0,0 +1,145 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spocontainer
+applicable: SharePoint Online
+title: Get-SPOAppBillingPolicies
+schema: 2.0.0
+author: arakesh
+ms.author: arakesh
+ms.reviewer:
+---
+
+# Get-SPOAppBillingPolicies
+
+## SYNOPSIS
+
+Returns billing policies that are owned by the tenant.
+
+## SYNTAX
+
+```powershell
+
+Get-SPOAppBillingPolicies
+```
+
+## DESCRIPTION
+
+Get-SPOAppBillingPolicies returns a list of billing policies that are owned by the tenant. If the tenant has no billing policies associated, the cmdlet will produce no output. 
+
+Each billing policy will include information such as Application Id, Azure subscription ID, Resource Group, Region, State of the subscription and the usage charging model.
+
+You must be a SharePoint Online Administrator or Global Administrator to run this cmdlet.
+
+> [!NOTE]
+> To sign into SharePoint Online PowerShell Module use the below cmdlet for authentication
+> 
+```powershell
+Connect-SPOService -Url https://contoso-admin.sharepoint.com

```

Comment:
>https://contoso-admin.sharepoint.com [](http://example.com/codeflow?start=24&length=36)

I think we should provide this contoso URL only in examples, otherwise it makes it look like this is the exact thing needed to be passed in all cases

Reply:
I changed the URL to https://(your-tenant)-admin.sharepoint.com and added a comment "Replace (your-tenant) with your actual SharePoint Online domain." Please let me know if that makes more sense.